tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -161,10 +161,10 @@ Standard output:
 @uifabric/styling: [XX:XX:XX XM] ■ Executing: /usr/local/bin/node "/office-ui-fabric-react/node_modules/typescript/lib/tsc.js" --pretty --target es5 --inlineSources --sourceRoot "../src" --outDir lib-amd --module amd --project "/office-ui-fabric-react/packages/styling/tsconfig.json"
 @uifabric/styling: [XX:XX:XX XM] ■ Running Jest
 @uifabric/styling: [XX:XX:XX XM] ■ /usr/local/bin/node "/office-ui-fabric-react/node_modules/jest/bin/jest.js" --config "/office-ui-fabric-react/packages/styling/jest.config.js" --passWithNoTests --colors
+@uifabric/styling:  PASS  src/styles/theme.test.ts
```

Comment:
@weswigham think this is a threading artifact?

Reply:
Probably - while I can make lerna/rush run serially (and do), there's no equivalent way to make gulp/just run serially, which is why I opened https://github.com/gulpjs/now-and-later/pull/15 (which is ultimately the library that both `gulp` and `just` use to kickoff concurrent operations) - so if a specific package does something like, say, kickoff its tests and webpack at the same time, there's not much I can do... I could strip all likes that say `webpack` from the output?

Reply:
@weswigham you can make gulp run serially with `gulp --series task1 task2 task3`. Is that what you are looking for?

Reply:
That only serializes the tasks listed on launch, not the internal `parallel` dependencies of some root task. And, to boot, since lerna/rush are orchestrating the builds, I don't even have access to the command line that's launching `gulp`. :v

Reply:
Awkward 😛 The config file should help with that last part but why can't the build be changed to serial?

Reply:
Because these are the live build/test/whatever tasks as defined by these projects. I'm not even usually invoking `gulp`, I'm usually just running `lerna build` or `rush rebuild` (both of which can be passed a flag to make _them_ execute serially).

Moreover, even if I could mangle the commands `rush` or `lerna` invoked in a maintainable way, the individual packages being invoked don't usually have a `series`-based variant of their build tasks set up (they have no intrinsic need to).

I suppose I could write a `.gulp.js` file into the home directory (so it affects all projects), and, rather than exporting any config (since none of the options really do what I need), utilize the opportunity to inject code into the build to patch the require cache with a method looking for future and past imports matching `undertaker` and forcibly replace the `parallel` method with the `series` method wholesale, but that seems both hacky and brittle (and may not work if the methods have been bound and plucked off, as they usually are...).